test/metabase/test_setup.clj

```diff
@@ -99,3 +99,17 @@
   []
   (log/info "Shutting down Metabase unit test runner")
   (core/stop-jetty!))
+
+(defn call-with-test-scaffolding
```

Comment:
in that case I guess it doesn't really make a big difference, you can leave it public